framework/src/application/node/node.js

```diff
@@ -330,6 +330,54 @@ module.exports = class Node {
 			blockTime: this.options.constants.BLOCK_TIME,
 		});
 
+		this.chain.events.on('NEW_BLOCK', eventData => {
+			const { block } = eventData;
+			// Publish to the outside
+			this.channel.publish('app:newBlock', eventData);
+
+			// Remove any transactions from the pool on new block
+			if (block.transactions.length) {
+				this.transactionPool.onConfirmedTransactions(
+					block.transactions.map(tx => this.chain.deserializeTransaction(tx)),
+				);
+			}
+
+			if (!this.synchronizer.isActive && !this.rebuilder.isActive) {
```

Comment:
`this.rebuilder.isActive` seems like not used properly in the rebuilder.js. Can you update this variable to be true inside of rebuild()?